### PR TITLE
What the database

### PR DESCRIPTION
**Pull request by StoneCat**
Hope that i can end this at weekend

-> merge because showing the pdf our supervisor

### CODE REVIEW

File: thesis-de/content-de.tex [hunk 1]
```diff
@@ -393,33 +393,143 @@ \subsubsection{Durchführung der Evaluierung}
 Datenhaltungssystem handelt, sondern um ein Werkzeug zur Datenanalyse,
 Datenaufbereitung und Datendarstellung.
 
-Bei der Evaluation von den anderen Datenhaltungssystemen konnten folgende
-Ergebnisse ermittelt werden:
-\begin{outline}
-  \1 elasticsearch
+Die daraus risultierenden Ergebnisse wurden Herrn Luis anschließend von den
```
Comment (bastelfreak) [inline thesis-de/content-de.tex]: resultierenden

File: thesis-de/content-de.tex [hunk 2]
```diff
@@ -393,33 +393,143 @@ \subsubsection{Durchführung der Evaluierung}
...
+Für die Anbindung späterer Systeme und Methodiken der Datenanalyse, wird 
+``Hadoop Common'' eingesetzt. Es handelt sich dabei um ein reines
+Schnitstellen-Modul, welches standartisierte Protokolle implementiert und
+betreibt. Es ermöglicht zum Beispiel die Anbindung der
+Software ``Hive'' und ``Impala'', welche in den Punkt
+~\ref{subsubsubsubsec:hadoop_beschreibung} und 
+~\ref{subsubsubsubsec:impala_beschreibung} erläutert werden.
+
+Für die Bearbeitung und Analyse von zeitkritischen Daten kommt die
+Komponente ``Hadoop Map Reduce'' zum Einsatz. Es handelt sich dabei um
```
Comment (bastelfreak) [inline thesis-de/content-de.tex]: Map Reduce ist (auch) ein Programmiermodell

**Review by bastelfreak — CHANGES_REQUESTED** [review]

File: thesis-de/content-de.tex (hunk 1 above)
Comment (StoneCat) [reply thesis-de/content-de.tex]: https://github.com/DCC-org/documents/pull/80/commits/1fe304d00f3be109277d127571f7127fd731a93d

File: thesis-de/content-de.tex (hunk 2 above)
Comment (StoneCat) [reply thesis-de/content-de.tex]: https://github.com/DCC-org/documents/pull/80/commits/1fe304d00f3be109277d127571f7127fd731a93d

**Review by bastelfreak — APPROVED** [review]

**Pull request merged**